Server/mods/deathmatch/logic/CGame.h

```diff
@@ -665,7 +665,12 @@ class CGame
 
     int m_iMaxClientTriggeredEventsPerInterval = 100;
     int m_iClientTriggeredEventsIntervalMs = 1000;
-    long long m_lClientTriggeredEventsLastCheck = 0;
 
-    std::map<CPlayer*, int> m_mapClientTriggeredEvents;
+    struct ClientTriggeredEventsInfo
+    {
+        long long m_llTicks;
```

Comment:
One more thing. I know that std::map::operator[] gives you a value-initialized object and fields are filled with zero. But there can be a situation when you wants to create an object of ClientTriggeredEventsInfo manually. And probably you will forget that those fields are not initialized by default. And this will be a problem. So I recommend you to initialize fields by default values.